index.html

```diff
@@ -8,6 +8,16 @@
     <link rel="stylesheet" href="/dist/breakpoint.css">
 </head>
 <body>
-    <h1>Content goes here</h1>
+    <div class="container">
+        <h1>Content goes here</h1>
+        <div class="grid">
+            <div class="item">1</div>
+            <div class="item">2</div>
+            <div class="item">3</div>
+            <div class="item">4</div>
+            <div class="item">5</div>
+            <div class="item">6</div>
+        </div>
+    </div>
 </body>
 </html>
```

Comment:
I think you need to delete this file, its in the gitignore but still checked in to git